vcx/wrappers/node/src/rustlib.ts

```diff
@@ -23,14 +30,14 @@ export const FFI_CONNECTION_DATA = 'string'
 export const FFI_VOID = ref.types.void
 export const FFI_CONNECTION_HANDLE_PTR = ref.refType(FFI_CONNECTION_HANDLE)
 export const FFI_CALLBACK_PTR = 'pointer'
-export const FFI_COMMAND_HANDLE = 'uint32'
-export const FFI_CREDENTIAL_HANDLE = 'uint32'
-export const FFI_PROOF_HANDLE = 'uint32'
-export const FFI_CREDENTIALDEF_HANDLE = 'uint32'
-export const FFI_SCHEMA_HANDLE = 'uint32'
-export const FFI_SCHEMA_NUMBER = 'uint32'
-export const FFI_PAYMENT_HANDLE = 'uint32'
-export const FFI_PRICE = 'uint32'
+export const FFI_COMMAND_HANDLE = FFI_UINT
+export const FFI_CREDENTIAL_HANDLE = FFI_UINT
```

Comment:
I'm not familiar with whole VCX API, but I think current changes may be incorrect for some part of API.
Like `vcx_issuer_credential_serialize` define u32 signature on public C API endpoint.

Reply:
@jovfer You might be right. I think to fix the particular issue which I've had, it's probably sufficient to do this change only for `FFI_UNSIGNED_INT_PTR`.
I can test it and if it works, I'll update pull request with this change only for the `FFI_UNSIGNED_INT_PTR`.

Reply:
@Patrik-Stas could you double-check all these constants and compare with original rust API?

I think proposal in your comment is correct: only `FFI_UNSIGNED_INT_PTR` should be fixed and probably there are some misuse or missed usage of it (at lease I can see one in `vcx/wrappers/node/src/rustlib.ts` `vcx_connection_sign_data` )